### ISSUE TITLE
Vote page: marry L2 proposal data with L1 proposal data

### ISSUE THREAD

**Issue by wildmolasses** [ScopeLift/l2-flexible-voting-frontend#45]
Show some kind of indicator on whether the proposal has bridged or not.
Link to the L1 and L2 proposal on tally, respectively

**Comment by wildmolasses** [comment]
data structure brainstorm; still gotta look into some of this

```ts
type Proposal = {
  proposalId: number; // same on both chains
  status: {
    l1: "Executed" | "Active" | ... ;
    l2: "Executed" | "Active" | ... | "Not bridged" // has the proposal been bridged to L2?
  votePower: { 
    l1: BigInt;
    l2: BigInt || undefined;
  description: string; // should come from L1
  startBlock: number;
  endBlock: number;
  }
}
```

**Comment by wildmolasses** [comment]
resolved by #64
